frontend/simple/views/Join.vue

```diff
@@ -153,11 +158,15 @@ export default {
       }
     }
   },
+  computed: {
+    contract () {
+      return this.$store.state[this.$route.query.groupId]
+    }
```

Comment:
Two problems here:

1. Name conflict with .vue internal state variable, `contract` (see below)
2. A computed state probably shouldn't be based on a query variable